netket/machine/abstract_machine.py

```diff
@@ -65,6 +65,33 @@ def der_log(self, x, out=None):
             """
         return NotImplementedError
 
+    def to_array(self, normalize=True, b_size=10000):
```

Comment:
maybe a deafult b_size=512 or something in that range would be more appropriate ?

Reply:
That is a good point, taking a power of 2 as the batch size speeds up the function.  I noticed that it does not really matter much which power you take.